src/main/archetype/ui.tests/test-module/specs/aem/sites.js

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright 2020 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+const path = require('path');
+const config = require('../../lib/config');
+const commons = require('../../lib/commons');
+const { expect } = require('chai');
+
+
+const AEM_SITES_PATH = '/sites.html';
+const AEM_SAMPLE_PAGE_PARENT = '/content/${appId}/us';
```

Comment:
@bpauli, is it possible to get this file parsing the actual value of `appId`?

Reply:
Forget this. I tested it and variable is already being replaced in created project.